package/yast2-auth-server.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jul 27 00:50:39 UTC 2022 - William Brown <william.brown@suse.com>
+
+- Version 4.5.1
+- Remove kerberos ldap database support
```

Comment:
We need a Bugzilla / Jira / Github reference here. If not, our CI tooling will complain at the time to send the OBS / IBS request like already happened in the latest yast-auth-client PR (see https://ci.opensuse.org/job/yast-yast-auth-client-master/32/console)